libraries/adaptive-expressions/webpack.config.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const defaultTSConfig = require('./tsconfig.json');
+
+module.exports = (env, argv) => {
+    return {
+        mode: 'none',
+
+        entry: {
+            index: './src/index.ts',
+        },
+        output: {
+            path: path.resolve(__dirname, './dist'),
+            libraryTarget: 'amd',
+            filename: 'index.js',
+        },
+        devtool: 'none',
+        resolve: {
+            extensions: ['.ts', '.js'],
+        },
+        module: {
+            rules: [
+                {
+                    test: /\.tsx?$/,
+                    loader: 'ts-loader',
+                    options: {
+                        compilerOptions: {
+                            ...defaultTSConfig.compilerOptions,
+                            outDir: './dist',
+                            target: 'es5',
+                        },
+                    },
+                    exclude: /node_modules/,
```

Comment:
I believe a safe-list would be better than instead of excluding `node_modules`. The entry is already `'./src/index.ts'` and a safe-list provides a better "locked-down" mechanism that ensures we only ship and webpack what we mean to.
```suggestion
                    include: [
                        path.resolve(__dirname, './src')
                    ],
```